.github/workflows/coverity.yml

```diff
@@ -10,6 +10,7 @@ on:
   workflow_dispatch:
 jobs:
   coverity:
+    if: github.repository == 'net-snmp/net-snmp'
```

Comment:
Does this test work as intended? The coverity.yml action still runs on my own github repository as one can see here: https://github.com/bvanassche/net-snmp/actions/runs/6647844856/job/18063944527